frontend/src/app/corpus-definitions/form/corpus-form/corpus-form.component.html

```diff
@@ -1,14 +1,35 @@
 <div class="section">
-    <div class="container">
+    <div class="container is-readable">
         <nav aria-label="secondary navigation" class="breadcrumb block">
             <ul>
-                <li><a [routerLink]="['/']">Corpora</a></li>
-                <li><a [routerLink]="['../..']">Definitions</a></li>
+                <li><a [routerLink]="['/']">I-Analyzer</a></li>
+                <li><a [routerLink]="['../..']">Custom corpora</a></li>
                 <li class="is-active" aria-current="page"><a>{{(corpus$ | async).definition?.meta.title ||
                         '...'}}: edit</a></li>
             </ul>
         </nav>
 
+        <div class="columns">
+            @let corpus = corpus$ | async;
+            <div class="column">
+                <h1 class="title">Edit "{{corpus?.definition?.meta?.title || '...'}}"</h1>
```

Comment:
[Elvis is alive!](https://en.wikipedia.org/wiki/Elvis_Presley_death_conspiracy_theories)